packages/@webex/plugin-meetings/src/reachability/index.ts

```diff
@@ -430,6 +456,29 @@ export default class Reachability {
     });
   }
 
+  /**
+   * Set client media IPs
+   * @param {RTCPeerConnection} peerConnection
+   * @param {string} clientMediaIP
+   * @returns {void}
+   */
+  private setClientMediaIPs(peerConnection: RTCPeerConnection, clientMediaIP?: string) {
+    const {CLOSED} = CONNECTION_STATE;
+
+    if (peerConnection.connectionState === CLOSED) {
+      LoggerProxy.logger.log(
+        `Reachability:index#setClientMediaIPs --> Attempting to set clientMediaIP of ${clientMediaIP} on closed peerConnection.`
+      );
+    }
+
+    // Set to null in case this fired from
+    // an event other than onIceCandidate
+    peerConnection.onicecandidate = null;
+    peerConnection.close();
```

Comment:
this is not needed here, it's done in setLatencyAndClose because setLatencyAndClose() is supposed to be the last thing that is called and closes the connection

Reply:
removed